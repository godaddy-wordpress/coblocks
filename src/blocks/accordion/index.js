/**
 * External dependencies
 */
import { AccordionIcon as icon } from '@godaddy-wordpress/coblocks-icons';

/**
 * Internal dependencies
 */
import edit from './edit';
import metadata from './block.json';
import transforms from './transforms';

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';
import { Icon } from '@wordpress/components';
import { InnerBlocks } from '@wordpress/block-editor';

/**
 * Block constants
 */
const { name, category, attributes } = metadata;

const settings = {
	attributes,
	/* translators: block description */
	description: __( 'Organize content within collapsable accordion items.', 'coblocks' ),
	edit,
	example: {
		attributes: {
			count: 2,
		},
	},
	icon: <Icon icon={ icon } />,
	keywords: [
		'coblocks',
		/* translators: block keyword */
		__( 'tabs', 'coblocks' ),
		/* translators: block keyword (abbreviation for "frequently asked questions") */
		__( 'faq', 'coblocks' ),
	],
	save() {
		return (
			<div>
				<InnerBlocks.Content />
			</div>
		);
	},
	supports: {
		align: [ 'wide', 'full' ],
		html: false,
	},
	/* translators: block name */
	title: __( 'Accordion', 'coblocks' ),
	transforms,
};

export { name, category, metadata, settings };
