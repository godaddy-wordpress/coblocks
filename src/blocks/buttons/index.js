/**
 * THIS BLOCK IS DEPRECRATED. IT WILL NOT SHOW UP IN THE INSERTER ANYMORE.
 * SEE /src/js/deprecations/deprecate-coblocks-buttons.js
 */

/**
 * Internal dependencies
 */
import deprecated from './deprecated';
import edit from './edit';
import metadata from './block.json';
import save from './save';
import transforms from './transforms';

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';
import { button as icon } from '@wordpress/icons';

/**
 * Block constants
 */
const { name, category, attributes } = metadata;

const settings = {
	/* translators: block name */
	title: __( 'Buttons', 'coblocks' ),
	/* translators: block description */
	description: __( 'Prompt visitors to take action with multiple buttons, side by side.', 'coblocks' ),
	icon,
	keywords: [
		'coblocks',
		/* translators: block keyword */
		__( 'link', 'coblocks' ),
		/* translators: block keyword (abbreviation for "call to action") */
		__( 'cta', 'coblocks' ),
		/* translators: block keyword */
		__( 'call to action', 'coblocks' ),
	],
	attributes,
	transforms,
	edit,
	save,
	deprecated,
};

export { name, category, metadata, settings };
