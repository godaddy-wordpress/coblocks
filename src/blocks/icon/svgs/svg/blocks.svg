<svg fill="none" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="m10.3519669 11.7141378v8.2858622h8.2815735v-8.2858622zm-10.3519669 8.2858622h8.2815735v-8.2858622h-8.2815735zm0-18.64319006v8.28586225h8.2815735v-8.28586225zm14.1407867-1.35680994-5.8592132 5.85189021 5.8592132 5.86224759 5.8592133-5.86224759z" fill="currentColor" /></svg>