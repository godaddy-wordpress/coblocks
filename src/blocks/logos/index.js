/**
 * External dependencies
 */
import { LogosIcon as icon } from '@godaddy-wordpress/coblocks-icons';

/**
 * Internal dependencies
 */
import deprecated from './deprecated';
import edit from './edit';
import metadata from './block.json';
import save from './save';

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';
import { Icon } from '@wordpress/components';

/**
 * Block constants.
 */
const { name, category, attributes } = metadata;

const settings = {
	/* translators: block name */
	title: __( 'Logos', 'coblocks' ),
	/* translators: block description */
	description: __( 'Add logos, badges, or certifications to build credibility.', 'coblocks' ),
	icon: <Icon icon={ icon } />,
	keywords: [
		'coblocks',
		/* translators: block keyword */
		__( 'clients', 'coblocks' ),
		/* translators: block keyword */
		__( 'proof', 'coblocks' ),
		/* translators: block keyword */
		__( 'testimonials', 'coblocks' ),
		/* translators: block keyword */
		__( 'awards', 'coblocks' ),
		/* translators: block keyword */
		__( 'seal', 'coblocks' ),
	],
	styles: [
		{
			name: 'default',
			/* translators: block style */
			label: __( 'Default', 'coblocks' ),
			isDefault: true,
		},
		{
			name: 'grayscale',
			/* translators: block style */
			label: __( 'Grayscale', 'coblocks' ),
		},
		{
			name: 'black-and-white',
			/* translators: block style */
			label: __( 'Black & White', 'coblocks' ),
		},
	],
	supports: {
		align: [ 'wide', 'full' ],
	},
	example: {
		attributes: {
			align: 'full',
			images: [
				{ url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/09/Wordpress-Logo.svg/600px-Wordpress-Logo.svg.png', width: 420 },
				{ url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/ae/WordPress.svg/640px-WordPress.svg.png', width: 340 },
			],
		},
	},
	attributes,
	edit,
	save,
	deprecated,
};

export { name, category, metadata, settings };
