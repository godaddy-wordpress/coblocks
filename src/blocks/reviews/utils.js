/**
 * External dependencies
 */
import { StarStyles } from '@godaddy-wordpress/coblocks-icons';

/**
 * WordPress dependencies
 */
import { Icon } from '@wordpress/components';

export const renderRating = ( rating, baseRating = 5 ) => {
	const ui = [];

	ui.push( ...Array( rating ).fill( renderRatingIcon( StarStyles.default ) ) );

	if ( baseRating - rating > 0 ) {
		ui.push( ...Array( baseRating - rating ).fill( renderRatingIcon( StarStyles.outlined ) ) );
	}

	return ui;
};

const renderRatingIcon = ( ratingIcon ) => <Icon icon={ ratingIcon } size={ 16 } />;
