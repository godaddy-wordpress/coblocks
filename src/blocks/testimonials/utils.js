/**
 * External dependencies.
 */
import { find } from 'lodash';

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';
import TokenList from '@wordpress/token-list';

export const blockStyles = [
	{
		isDefault: true,
		/* translators: block style */
		label: __( 'Boxy', 'coblocks' ),
		name: 'tall',
	},
	{
		/* translators: block style */
		label: __( 'Conversation', 'coblocks' ),
		name: 'conversation',
	},
	{
		/* translators: block style */
		label: __( 'Horizontal', 'coblocks' ),
		name: 'horizontal',
	},
];

/**
 * Returns the active style from the given className.
 *
 * @param {Array}  styles    Block style variations.
 * @param {string} className Class name
 * @return {Object?} The active style.name.
 */
export const getActiveStyle = ( styles, className ) => {
	for ( const style of new TokenList( className ).values() ) {
		if ( style.indexOf( 'is-style-' ) === -1 ) {
			continue;
		}

		const potentialStyleName = style.substring( 9 );
		const activeStyle = find( styles, { name: potentialStyleName } );

		if ( activeStyle ) {
			return activeStyle.name;
		}
	}

	return find( styles, 'isDefault' ).name;
};
