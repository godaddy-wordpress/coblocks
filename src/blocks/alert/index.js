/**
 * Internal dependencies
 */
import metadata from './block.json';

/**
 * WordPress dependencies
 */
import { createBlock } from '@wordpress/blocks';
import { dispatch } from '@wordpress/data';

/**
 * Block constants
 */
const { name } = metadata;

const settings = {
	edit: ( props ) => {
		const { replaceBlocks } = dispatch( 'core/block-editor' );

		replaceBlocks(
			[ props.clientId ],
			createBlock( 'core/paragraph', props.attributes ),
		);
		return null;
	},
	parent: [],
	save: () => null,
	title: metadata.title,
};

export { name, metadata, settings };
