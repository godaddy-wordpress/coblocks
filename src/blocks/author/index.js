/**
 * Internal dependencies
 */
import metadata from './block.json';

/**
 * WordPress dependencies
 */
import { createBlock } from '@wordpress/blocks';
import { dispatch, select } from '@wordpress/data';

/**
 * Block constants
 */
const { name } = metadata;

const settings = {
	edit: ( props ) => {
		const parentBlock = select( 'core/editor' ).getBlocksByClientId( props.clientId )[ 0 ];
		if ( ! parentBlock ) {
			return null;
		}

		// Used to set text color props
		const hasTextColor = ( props.attributes.hasOwnProperty( 'textColor' ) );
		const hasCustomTextColor = ( props.attributes.hasOwnProperty( 'style' ) &&
			props.attributes.style.hasOwnProperty( 'color' ) && props.attributes.style.color.hasOwnProperty( 'text' ) );

		// Used to set background color props
		const hasBackgroundColor = ( props.attributes.hasOwnProperty( 'backgroundColor' ) );
		const hasCustomBackgroundColor = ( props.attributes.hasOwnProperty( 'style' ) &&
			props.attributes.style.hasOwnProperty( 'color' ) && props.attributes.style.color.hasOwnProperty( 'background' ) );

		// Used to set font props
		const hasFontSize = ( props.attributes.hasOwnProperty( 'fontSize' ) );
		const hasCustomFont = ( props.attributes.hasOwnProperty( 'style' ) &&
			props.attributes.style.hasOwnProperty( 'typography' ) );

		const { replaceBlocks } = dispatch( 'core/block-editor' );
		const isRTL = select( 'core/editor' ).getEditorSettings().isRTL;

		const authorNameBlock = () => {
			let authorNameBlockProps = {
				content: props.attributes.name,
				level: 4,
			};

			// Font controls - default is fontSize and style is used for custom setting.
			// All font controls should apply to the bio and name.
			if ( hasFontSize ) {
				authorNameBlockProps = { ...authorNameBlockProps,
					fontSize: props.attributes.fontSize,
				};
			}
			if ( hasCustomFont ) {
				authorNameBlockProps = { ...authorNameBlockProps,
					style: { ...authorNameBlockProps?.style,
						typography: props.attributes.style.typography,
					},
				};
			}

			// Headings do not inherit color from parent - we descend the style here for initial match.
			if ( hasCustomTextColor ) {
				authorNameBlockProps = { ...authorNameBlockProps,
					style: { ...authorNameBlockProps?.style,
						color: { ...authorNameBlockProps?.style?.color,
							text: props.attributes.style.color.text,
						},
					},
				};
			}
			if ( hasTextColor ) {
				authorNameBlockProps = { ...authorNameBlockProps,
					textColor: props.attributes.textColor,
				};
			}

			return createBlock( 'core/paragraph', authorNameBlockProps );
		};

		const authorBioBlock = () => {
			let authorBioBlockProps = {
				content: props.attributes.biography,
			};

			// Font controls - default is fontSize and style is used for custom setting.
			// All font controls should apply to the bio and name.
			if ( hasFontSize ) {
				authorBioBlockProps = {
					...authorBioBlockProps,
					fontSize: props.attributes.fontSize,
				};
			}
			if ( hasCustomFont ) {
				authorBioBlockProps = {
					...authorBioBlockProps,
					style: {
						...authorBioBlockProps?.style,
						typography: props.attributes.style.typography,
					},
				};
			}

			return createBlock( 'core/paragraph', authorBioBlockProps );
		};

		const buttonBlock = parentBlock.innerBlocks;
		// Set buttons to align.
		buttonBlock.forEach( function( button, index ) {
			buttonBlock[ index ].attributes.align = isRTL ? 'right' : 'left';
		} );
		// Button blocks are no-longer stand-alone and should be within `core/buttons` block.
		const coreButtons = createBlock( 'core/buttons', { }, [ ...buttonBlock ] );

		const imageBlock = createBlock( 'core/image', { className: 'is-style-rounded', id: props.attributes.imgId, url: props.attributes.imgUrl } );

		const leftColumn = createBlock( 'core/column', { width: '25%' }, [ imageBlock ] );
		const rightColumn = createBlock( 'core/column', { width: '75%' }, [ authorNameBlock(), authorBioBlock(), coreButtons ] );

		const columnsBlock = () => {
			let columnsBlockProps = {
				...( props.attributes.hasOwnProperty( 'className' ) && { className: props.attributes.className } ),
				...( props.attributes.hasOwnProperty( 'animation' ) && { animation: props.attributes.animation } ),
				...( props.attributes.hasOwnProperty( 'anchor' ) && { anchor: props.attributes.anchor } ),
				style: {
					spacing: {
						padding: {
							bottom: '2.5rem',
							left: '2.5rem',
							right: '2.5rem',
							top: '2.5rem',
						},
					},
				},
			};

			// Migrate text colors into columns block props.
			if ( hasCustomTextColor ) {
				columnsBlockProps = { ...columnsBlockProps,
					style: { ...columnsBlockProps?.style,
						color: { ...columnsBlockProps?.style?.color,
							text: props.attributes.style.color.text,
						},
					},
				};
			}
			if ( hasTextColor ) {
				columnsBlockProps = { ...columnsBlockProps,
					textColor: props.attributes.textColor,
				};
			}

			// Migrate background colors into columns block props.
			if ( hasCustomBackgroundColor ) {
				columnsBlockProps = { ...columnsBlockProps,
					style: { ...columnsBlockProps?.style,
						color: { ...columnsBlockProps?.style?.color,
							background: props.attributes.style.color.background,
						},
					},
				};
			}
			if ( hasBackgroundColor ) {
				columnsBlockProps = { ...columnsBlockProps,
					backgroundColor: props.attributes.backgroundColor,
				};
			}

			// Does not have background color set. Use default block color from CoBlocks.
			const hasNoBackgroundColorSet = ! props.attributes.hasOwnProperty( 'backgroundColor' ) &&
				! props.attributes?.style?.color?.hasOwnProperty( 'background' );
			if ( hasNoBackgroundColorSet ) {
				// $dark-opacity-light-200 computed #8C8C971A - no variable access in root.
				columnsBlockProps = { ...columnsBlockProps,
					style: { ...columnsBlockProps?.style,
						color: { ...columnsBlockProps?.style?.color,
							background: '#8C8C971A',
						},
					},
				};
			}

			return createBlock( 'core/columns', columnsBlockProps, [ leftColumn, rightColumn ] );
		};

		replaceBlocks(
			[ props.clientId ],
			columnsBlock()
		);

		return null;
	},
	parent: [],
	save: () => null,
	/* translators: block name */
	title: metadata.title,
};

export { name, metadata, settings };
