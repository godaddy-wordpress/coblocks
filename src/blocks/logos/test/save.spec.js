/**
 * External dependencies
 */
import '@testing-library/jest-dom/extend-expect';
import { createBlock, registerBlockType, serialize } from '@wordpress/blocks';
import { replaceActiveStyle } from '@wordpress/block-editor/build/components/block-styles/utils';

/**
 * Internal dependencies.
 */
import { name, settings } from '../index';

// Make variables accessible for all tests.
let block;
let serializedBlock;

describe( 'coblocks/logos', () => {
	beforeAll( () => {
		// Register the block.
		registerBlockType( name, { category: 'common', ...settings } );
	} );

	beforeEach( () => {
		// Create the block with the minimum attributes.
		block = createBlock( name );

		// Reset the reused variables.
		serializedBlock = '';
	} );

	it( 'should render with images attribute', () => {
		block.attributes.images = [
			{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg', id: 1 },
		];
		serializedBlock = serialize( block );

		expect( serializedBlock ).toBeDefined();
		expect( serializedBlock ).toContain( 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg' );
		expect( serializedBlock ).toContain( 'data-id="1"' );

		expect( serializedBlock ).toMatchSnapshot();
	} );

	it( 'should render content align classes', () => {
		[ 'wide', 'full' ].forEach( ( alignment ) => {
			block.attributes.align = alignment;
			block.attributes.images = [
				{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg', id: 1 },
				{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-2.jpg', id: 2 },
				{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-3.jpg', id: 3 },
				{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-4.jpg', id: 4 },
				{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-5.jpg', id: 5 },
			];
			serializedBlock = serialize( block );
			expect( serializedBlock ).toBeDefined();
			expect( serializedBlock ).toContain( 'align' + alignment );
			expect( serializedBlock ).toMatchSnapshot();
		} );
	} );

	it( 'should render with style "Black & White"', () => {
		block.attributes.className = '';
		block.attributes.images = [
			{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg', id: 1 },
		];

		const activeStyle = undefined;
		const newStyle = { name: 'black-and-white' };
		block.attributes.className = replaceActiveStyle( block.attributes.className, activeStyle, newStyle );

		serializedBlock = serialize( block );

		expect( serializedBlock ).toBeDefined();
		expect( serializedBlock ).toContain( 'is-style-black-and-white' );

		expect( serializedBlock ).toMatchSnapshot();
	} );

	it( 'should render with style "Grayscale"', () => {
		block.attributes.className = '';
		block.attributes.images = [
			{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg', id: 1 },
		];

		const activeStyle = undefined;
		const newStyle = { name: 'grayscale' };
		block.attributes.className = replaceActiveStyle( block.attributes.className, activeStyle, newStyle );

		serializedBlock = serialize( block );

		expect( serializedBlock ).toBeDefined();
		expect( serializedBlock ).toContain( 'is-style-grayscale' );

		expect( serializedBlock ).toMatchSnapshot();
	} );

	it( 'should render with images.width attribute', () => {
		block.attributes.images = [
			{ url: 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg', id: 1, width: '60.6897%' },
		];
		serializedBlock = serialize( block );

		expect( serializedBlock ).toBeDefined();
		expect( serializedBlock ).toContain( 'https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg' );
		expect( serializedBlock ).toContain( 'data-width="60.6897%"' );
		expect( serializedBlock ).toContain( 'style="width:60.6897%"' );

		expect( serializedBlock ).toMatchSnapshot();
	} );

	it( 'should render with className attribute', () => {
		block.attributes.className = 'my-custom-class';
		serializedBlock = serialize( block );

		expect( serializedBlock ).toBeDefined();
		expect( serializedBlock ).toContain( '"className":"my-custom-class"' );
		expect( serializedBlock ).toContain( 'my-custom-class' );

		expect( serializedBlock ).toMatchSnapshot();
	} );
} );
