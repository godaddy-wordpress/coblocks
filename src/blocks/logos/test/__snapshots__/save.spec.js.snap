// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`coblocks/logos should render content align classes 1`] = `
"<!-- wp:coblocks/logos {\\"align\\":\\"wide\\"} -->
<div aria-label=\\"List of logos\\" class=\\"wp-block-coblocks-logos alignwide\\"><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:25%\\"><img data-id=\\"1\\" data-width=\\"25%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg\\"/></div><div style=\\"width:25%\\"><img data-id=\\"2\\" data-width=\\"25%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-2.jpg\\"/></div><div style=\\"width:25%\\"><img data-id=\\"3\\" data-width=\\"25%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-3.jpg\\"/></div><div style=\\"width:25%\\"><img data-id=\\"4\\" data-width=\\"25%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-4.jpg\\"/></div></div><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:100%\\"><img data-id=\\"5\\" data-width=\\"100%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-5.jpg\\"/></div></div></div>
<!-- /wp:coblocks/logos -->"
`;

exports[`coblocks/logos should render content align classes 2`] = `
"<!-- wp:coblocks/logos {\\"align\\":\\"full\\"} -->
<div aria-label=\\"List of logos\\" class=\\"wp-block-coblocks-logos alignfull\\"><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:20%\\"><img data-id=\\"1\\" data-width=\\"20%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg\\"/></div><div style=\\"width:20%\\"><img data-id=\\"2\\" data-width=\\"20%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-2.jpg\\"/></div><div style=\\"width:20%\\"><img data-id=\\"3\\" data-width=\\"20%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-3.jpg\\"/></div><div style=\\"width:20%\\"><img data-id=\\"4\\" data-width=\\"20%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-4.jpg\\"/></div><div style=\\"width:20%\\"><img data-id=\\"5\\" data-width=\\"20%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-5.jpg\\"/></div></div></div>
<!-- /wp:coblocks/logos -->"
`;

exports[`coblocks/logos should render with className attribute 1`] = `"<!-- wp:coblocks/logos {\\"className\\":\\"my-custom-class\\"} /-->"`;

exports[`coblocks/logos should render with images attribute 1`] = `
"<!-- wp:coblocks/logos -->
<div aria-label=\\"List of logos\\" class=\\"wp-block-coblocks-logos\\"><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:100%\\"><img data-id=\\"1\\" data-width=\\"100%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg\\"/></div></div></div>
<!-- /wp:coblocks/logos -->"
`;

exports[`coblocks/logos should render with images.width attribute 1`] = `
"<!-- wp:coblocks/logos -->
<div aria-label=\\"List of logos\\" class=\\"wp-block-coblocks-logos\\"><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:60.6897%\\"><img data-id=\\"1\\" data-width=\\"60.6897%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg\\"/></div></div></div>
<!-- /wp:coblocks/logos -->"
`;

exports[`coblocks/logos should render with style "Black & White" 1`] = `
"<!-- wp:coblocks/logos {\\"className\\":\\"is-style-black-and-white\\"} -->
<div aria-label=\\"List of logos\\" class=\\"wp-block-coblocks-logos is-style-black-and-white\\"><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:100%\\"><img data-id=\\"1\\" data-width=\\"100%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg\\"/></div></div></div>
<!-- /wp:coblocks/logos -->"
`;

exports[`coblocks/logos should render with style "Grayscale" 1`] = `
"<!-- wp:coblocks/logos {\\"className\\":\\"is-style-grayscale\\"} -->
<div aria-label=\\"List of logos\\" class=\\"wp-block-coblocks-logos is-style-grayscale\\"><div class=\\"wp-block-coblocks-logos__row\\"><div style=\\"width:100%\\"><img data-id=\\"1\\" data-width=\\"100%\\" src=\\"https://wordpress.com/wp-content/uploads/1234/56/image-1.jpg\\"/></div></div></div>
<!-- /wp:coblocks/logos -->"
`;
